bin/mob.js

```diff
@@ -141,6 +144,23 @@ function author() {
   return oneLine`${name} <${email}>`;
 }
 
+function printConfigWarning() {
+  const name = config.get('user.name');
+  const email = config.get('user.email');
```

Comment:
Since the git query for getting the name and email is done in the `author()` function, extract that query into another function and reuse the result for `printConfigWarning()`. To prevent the need to run the query twice.